mdb.py

```
@@ -226,14 +688,742 @@ def execute_change(db, server, sql):
+        logging.debug(f"Server: {server}")
+        logging.debug(f"SQL: {sql}")
+        logging.debug("=" * 80)
         dsn = get_config()['servers'][server]['dsn'][0]
-        cmd = ('mysql -h %s -P %s -u %s -p%s main   -e "%s" ' % (dsn['host'], dsn['port'], dsn['user'], dsn['passwd'], sql))
+
+        # Escape double quotes and backslashes in SQL to prevent shell interpretation issues
+        # This allows both single and double quotes to be used in SQL statements
+        escaped_sql = sql.replace('\\', '\\\\').replace('"', '\\"').replace('$', '\\$').replace('`', '\\`')
+
+        cmd = ('mysql -h %s -P %s -u %s -p%s main   -e "%s" ' % (dsn['host'], dsn['port'], dsn['user'], dsn['passwd'], escaped_sql))
         p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Ruff (0.15.4)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)